### ISSUE TITLE
Black screen while playing video

### ISSUE THREAD

**Issue by chiragtalaviya** [PierfrancescoSoffritti/android-youtube-player#1032]
### Description of the Bug
It shows black screen while playing video in full screen

### Environment details
Android Version: Android 13
API Version: 33

Tested devices:
- Samsung Galaxy Tab A7

Youtube Player Library Version: 12.0.0 

### Steps to reproduce the bug
1. Create an full screen activity with youtubeplayer
2. Start the application on the target device
3. Tap on the video player
4. See player displaying black screen

### Expected behavior
Should play video

### Actual behavior
Displaying black screen

**Comment by PierfrancescoSoffritti** [comment]
Please [read the doc](https://github.com/PierfrancescoSoffritti/android-youtube-player#full-screen) and have a look at the sample app. The fullscreen video plays in a separate view.

**Comment by chiragtalaviya** [comment]
I have already implemented it.
![WhatsApp Image 2023-07-25 at 7 45 06 PM](https://github.com/PierfrancescoSoffritti/android-youtube-player/assets/25198561/180d5519-6b6d-4546-a249-2a100827ad94)

**Comment by PierfrancescoSoffritti** [comment]
I suggest looking at the sample app, and copy the code from there.

Please share a snippet of how you're handling fullscreen.

**Comment by chiragtalaviya** [comment]
`lifecycle.addObserver(binding.youtubePlayerView)

        val iFramePlayerOptions: IFramePlayerOptions = IFramePlayerOptions.Builder()
            .controls(1)
            .fullscreen(1)
            .build()

        binding.youtubePlayerView.initialize(object : AbstractYouTubePlayerListener() {
            override fun onReady(youTubePlayer: YouTubePlayer) {
                this@FullScreenVideoActivity.youTubePlayer = youTubePlayer
                val videoId = "S0Q4gqBUs7c"
//                youTubePlayer.loadVideo("ghIxPREd2GI", 0f)
                youTubePlayer.loadVideo(Utils.extractYTId(videoLink), 0f)
            }
        }, true, iFramePlayerOptions)

        binding.youtubePlayerView.addFullscreenListener(object : FullscreenListener {
            override fun onEnterFullscreen(fullscreenView: View, exitFullscreen: Function0<Unit>) {
                isFullscreen = true

                // the video will continue playing in fullscreenView
                binding.youtubePlayerView.visibility = View.GONE
                binding.fullScreenViewContainer.visibility = View.VISIBLE
                binding.fullScreenViewContainer.addView(fullscreenView)

                requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
            }

            override fun onExitFullscreen() {
                isFullscreen = false

                // the video will continue playing in the player
                binding.youtubePlayerView.visibility = View.VISIBLE
                binding.fullScreenViewContainer.visibility = View.GONE
                binding.fullScreenViewContainer.removeAllViews()

                requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
            }
        })`

**Comment by PierfrancescoSoffritti** [comment]
What about your layout file?

**Comment by chiragtalaviya** [comment]
![image](https://github.com/PierfrancescoSoffritti/android-youtube-player/assets/25198561/0d02b947-b1c9-487b-97a5-ae8602b81079)
Here it is

**Comment by PierfrancescoSoffritti** [comment]
Does the fullscreen example work when you run directly the sample app?

**Comment by chiragtalaviya** [comment]
yes, even my player is working fine in other tablets but in Samsung tablet it showing black screen.

**Comment by Naguchennai** [comment]
Yes also having same issue some time black screen only ..video not playing

**Comment by PierfrancescoSoffritti** [comment]
So does the sample app work in your Samsung tablet?

It could be that that specific table has a modified version of Webview.

**Comment by Naguchennai** [comment]
i have issue with samsung s22 mobile

**Comment by PierfrancescoSoffritti** [comment]
Have you tried running the sample app?

**Comment by chiragtalaviya** [comment]
> 

Actually I don't have that tablet. This issue is reported by user.

**Comment by Naguchennai** [comment]
i think i missed

       @Override
    public void onDestroy() {
        super.onDestroy();
        youTubePlayerView.release();
    }

**Comment by zeeshanayaz** [comment]
In your AndroidManifest file, add this line in application tag. This might fix your issue
`android:hardwareAccelerated="true"`
